JitsiMeetJS.ts

```diff
@@ -86,6 +86,9 @@ interface IJitsiMeetJSOptions {
     flags?: {
         runInLiteMode?: boolean;
         ssrcRewritingEnabled?: boolean;
+    },
+    logging?: {
+        disableLogCollector?: boolean;
```

Comment:
I think there's an analytics option called rtcstatsStoreLogs, maybe we can reuse that?

Reply:
Hum ... https://github.com/jitsi/jitsi-meet/blob/59420ea1a8204201db0b3203818594ef4ac66462/config.js#L1807
I'm confused ... this will disable it in jitsi-meet, that's why I added it:
https://github.com/jitsi/jitsi-meet/blob/e5fa25892ec33bec4a87d0153c0d2273ef4e4bdf/react/features/base/logging/middleware.ts#L155

So we can disable it via two configs. So probably I need to check both, otherwise we can disable it via one place in jitsi-meet and still be sending logs ... 

WDYT?

Reply:
I will switch it to the other one.

Reply:
Done!